core/api/tests/test_cp_report_comments.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+pytestmark = pytest.mark.django_db
+
+
+class TestCPReportComments:
+    client = APIClient()
+    SECTION_A = "section_a"
+    SECTION_B = "section_b"
+    COMMENT_COUNTRY = "comment_country"
+    COMMENT_SECRETARIAT = "comment_secretariat"
+
+    def test_without_permission_secretariat(self, user, cp_report_2019):
+        url = reverse(
+            "country-programme-report-comments", kwargs={"id": cp_report_2019.id}
+        )
+
+        # try to create country comment
+        self.client.force_authenticate(user=user)
+        data = {
+            "section": self.SECTION_A,
+            "comment_type": self.COMMENT_COUNTRY,
+            "comment": "Test create country",
+        }
+        response = self.client.post(url, data, format="json")
+        assert response.status_code == 400
+
+    def test_without_permission_country(self, country_user, cp_report_2019):
+        url = reverse(
+            "country-programme-report-comments", kwargs={"id": cp_report_2019.id}
+        )
+
+        # try to create secretariat comment
+        self.client.force_authenticate(user=country_user)
+        data = {
+            "section": self.SECTION_A,
+            "comment_type": self.COMMENT_SECRETARIAT,
+            "comment": "Test create secretariat",
```

Comment:
lame comment... I suggest TopManeleDinToateTimpurile generator for test texts :)))